brainstormforce/ultimate-addons-for-gutenberg
# Fix: Price List - Fixed missing image padding option, Editor css class updated  and columns not updating issue.

### Description

- Price list IMG padding option missing. Also some of the settings not working in the editor.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Check image padding option, row gap and column gap is working in editor or not.
- Set image position left and add new child and add image and check image position on frontend.
- Set column count to 1 Update and refresh page and add new child and check last added child.

### Checklist:
